api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxNewCohortQueries.java

```diff
@@ -13,26 +13,28 @@
  */
 package org.openmrs.module.eptsreports.reporting.library.cohorts;
 
+import static org.openmrs.module.reporting.evaluation.parameter.Mapped.mapStraightThrough;
```

Comment:
there is `@Autowired private HivCohortQueries hivCohortQueries` that is not  being  used,so it should be removed at line  `52`